### PR TITLE
Add README_fr

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
No README_fr

## Solution
Add README_fr

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [x] **Approval (LGTM)** : Kay0u
- **CI succeeded** : 
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by kay0u** [comment]
Can be merged in 3 days

**Comment by maniackcrudelis** [comment]
Why do you remove those information ?

File: README_fr.md [hunk 1]
```diff
@@ -50,10 +42,8 @@ Jirafeau est un site web permettant d'héberger et de partager des fichiers de m
 
 ---
 
-Informations pour les développeurs
-----------------
+## Informations pour les développeurs
 
-**Seulement si vous voulez utiliser une branche de test pour le codage, au lieu de fusionner directement dans la banche principale.**
```
Comment (ericgaspar) [inline README_fr.md]: As I understand, this statement in bold is for the app packager and it makes little sense to leave it. The other entries are empty, therefore not needed (Limitations, Configuration, Additional information). The Badge for Jessie seems to be outdated.

Comment (maniackcrudelis) [reply README_fr.md]: This is for the contributors to the app, not for the packager.
Empty categories are still useful as it keep the template similar to all other apps. And so, if an info is added, it's easy to know where to put it.

Comment (ericgaspar) [reply README_fr.md]: I was told in a previous pull request for another app to remove this as it was only for packagers...

**Pull request merged**